.eslintrc.cjs

```diff
@@ -36,6 +36,8 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: lintTypes
     ? {
+        // this is not yet compatible with eslint lsp so it's conditioned on AGORIC_ESLINT_TYPES
```

Comment:
Recommend “gated”, “contingent on”, or “predicated”.

Reply:
all better, though imo not different enough to suffer through the _Chain deployment test_ again.